src/models/SqlOutputContentProvider.ts

```diff
@@ -475,7 +471,7 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
             }
         }
 
-        vscode.workspace.openTextDocument(uri).then((doc: vscode.TextDocument) => {
+        vscode.workspace.openTextDocument({ language: linkType }) .then((doc: vscode.TextDocument) => {
```

Comment:
In the current code I don't think we'll get here without a linkType of xml or json, since we only add the link for those types. But even if we somehow did have a different linkType, we would want to open the link with the default behavior, so I'll keep it as-is.